### PR TITLE
Make litex_json2renode work with default arty target SOC .json file

### PR DESCRIPTION
**Pull request by rasmuspeders1**
I ran into trouble trying to generate renode config from the json file generated with `python3 -m litex_boards.targets.digilent_arty`.

It simply crashes on various items.

I have worked around the issues to make the script complete and it now generates usable .repl and .resc files now.

### CODE REVIEW

File: litex/tools/litex_json2renode.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def get_cpu_type(csr):
     return (kind, variant)
 
 def get_cpu_count(csr):
-    return csr['constants']['config_cpu_count']
+    return csr['constants'].get('config_cpu_count',1)
```
Comment (mateusz-holenko) [inline litex/tools/litex_json2renode.py:217]: Small formatting suggetsion
```suggestion
    return csr['constants'].get('config_cpu_count', 1)
```

File: litex/tools/litex_json2renode.py [hunk 2]
```diff
@@ -889,8 +889,9 @@ def generate_resc(csr, number_of_cores, args, flash_binaries={}, tftp_binaries={
 sysbus LoadBinary @{} {}
 """.format(args.bios_binary, hex(opensbi_base))
 
-    for cpu_id in range(0, number_of_cores):
-        result += f"cpu{cpu_id} PC {hex(opensbi_base)}\n"
+    if opensbi_base:
```
Comment (mateusz-holenko) [inline litex/tools/litex_json2renode.py:892]: Could you split this change to a separate commit?

**Review by mateusz-holenko — COMMENTED** [review]
Some minor remarks, but generally looks good.

Thanks @rasmuspeders1 for your contribution!

File: litex/tools/litex_json2renode.py (hunk 2 above)
Comment (rasmuspeders1) [reply litex/tools/litex_json2renode.py:892]: done

File: litex/tools/litex_json2renode.py (hunk 1 above)
Comment (rasmuspeders1) [reply litex/tools/litex_json2renode.py]: done

**Comment by rasmuspeders1** [comment]
Thanks for the feedback. Your comments should have been addressed now.

**Review by mateusz-holenko — APPROVED** [review]
